dispatch/tests/test_api_pages.py

```diff
@@ -0,0 +1,239 @@
+from django.core.urlresolvers import reverse
+
+from rest_framework import status
+
+from dispatch.tests.cases import DispatchAPITestCase
+
+from dispatch.apps.content.models import Page, Person
```

Comment:
Sounds good where is Person used? From looking at the class, it has info for admin. Is this to differentiate the reporters vs. programmers, and what privileges do each have?